style.css

```diff
@@ -0,0 +1,41 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: hsl(41, 70%, 94%);
+  color: hsl(41, 70%, 20%);
+  font-family: "Libre Baskerville", serif;
+}
+
+h1 {
+  background-color: white;
+  color: rgb(31, 70, 122);
+  font-size: 40px;
+  text-align: center;
```

Comment:
If all the text on your page is centered, consider adding `text-align: center` to your `body` ruleset, which would allow you to eliminate it everywhere else.